ts/models/conversation.ts

```diff
@@ -2588,17 +2588,27 @@ export function hasValidOutgoingRequestValues({
   isBlocked,
   isPrivate,
   activeAt,
+  hasMessages,
 }: {
   isMe: boolean;
   isApproved: boolean;
   didApproveMe: boolean;
   isBlocked: boolean;
   isPrivate: boolean;
   activeAt: number;
+  hasMessages?: boolean;
 }): boolean {
   const isActive = activeAt && isFinite(activeAt) && activeAt > 0;
 
-  return Boolean(!isMe && isApproved && isPrivate && !isBlocked && !didApproveMe && isActive);
+  // Started a new message, but haven't sent a message yet
+  const emptyConvo =
```

Comment:
I'd rather not have a test based on messages being there or not, when messages can disappear

Reply:
This has been resolved by relying on the contact info from libsession.